tests/test_quantity.py

```diff
@@ -329,11 +329,11 @@ def test_complex_temperature_difference_to():
     m = ansunits.Quantity(1.0, "kg")
     result = m * (t2 - t1)
     resultC1 = result.to("kg C")
-    assert resultC1.type == "Temperature Difference"
+    assert resultC1.type == "Composite"
```

Comment:
If temp diff info is lost, can it be regained. E.g.,
```
result = m * (t2 - t1)
result = result / m
# what is result.type now?
```

Reply:
That result would be "Temperature" once the division or multiplication happens it is lost. When the dimensions are used to create a unit all dimension_order #4 are combined into one "Temperature" currently.

Reply:
I believe that the previous behaviour is also wrong, because temperature difference is being conflated with other things. If we want to track temperature difference, then we should do it properly. If we build it into the dimensions computation then we can also cancel temperature difference correctly. That isn't what we have been doing though; it's hacky.

Regarding the current change, the following is the scenario of concern:

```
t1 = Quantity(1.0, "K")
t2 = Quantity(2.0, "K")
m = Quantity(2.0, "kg")
result = m * (t2 - t1)
result = result / m
result = result.to("C") # or "deltaC" ???
```

Reply:
Yeah I agree that should be a temperature difference at the end